test/integration/apiserver/apply/reset_fields_test.go

```diff
@@ -56,6 +56,7 @@ var resetFieldsStatusData = map[schema.GroupVersionResource]string{
 	gvr("networking.k8s.io", "v1beta1", "ingresses"):                `{"status": {"loadBalancer": {"ingress": [{"ip": "127.0.0.2"}]}}}`,
 	gvr("networking.k8s.io", "v1", "ingresses"):                     `{"status": {"loadBalancer": {"ingress": [{"ip": "127.0.0.2"}]}}}`,
 	gvr("autoscaling", "v1", "horizontalpodautoscalers"):            `{"status": {"currentReplicas": 25}}`,
+	gvr("autoscaling", "v2", "horizontalpodautoscalers"):            `{"status": {"currentReplicas": 25}}`,
```

Comment:
I'm surprised v2beta1 and v2beta2 weren't already in this list

Reply:
I will try to add  v2beta1 and v2beta2 into this list.

Reply:
As if pull-kubernetes-integration test fails when I try to add v2beta1 and v2beta2 into this list. So I delete them from this list.